lib/extract_i18n/adapters/ruby_adapter.rb

```diff
@@ -122,6 +122,11 @@ def ignore_parent?(node)
       node.children[1] == :require ||
         node.type == :regexp ||
         (node.type == :pair && ExtractI18n.ignore_hash_keys.include?(node.children[0].children[0].to_s)) ||
+        (node.type == :pair &&
+          node.children[0].children[0].to_s == 'description' &&
+          # Look up to possible _t function call
+          @nesting[-4] && ignore_parent?(@nesting[-4])
+        ) ||
```

Comment:
i truly don't understand this code and tbh that seems fine. also i love this comment.